<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAQWidget</class>
 <widget class="QWidget" name="DAQWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAQWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Decode</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>FTP sync</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="inSetRawDir">
     <property name="text">
      <string>Raw Dir</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="inDataFolder">
     <property name="text">
      <string>N:/data/test_station/Demo/Data</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Interval:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="inInterval">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="QProgressBar" name="outFtpSyncPBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QProgressBar" name="outDecodePBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="inOpenFolder">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="outSize">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="inDecode">
     <property name="text">
      <string>Enable Decoding</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="inFtpSync">
     <property name="text">
      <string>Enable FTP Sync</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
